src/indigo/less/outer-pages.less

```diff
@@ -281,3 +281,234 @@
         }
     }
 }
+
+.splash {
+    &-1 {
+        &.splash-background {
+            @media (max-width: 899px) {
```

Comment:
@janmichek nemal by byt tento najmensi obrazok bez media queries? teda ak riesim aspon trosku mobile-first.

Reply:
Ano, dobrej postreh, melo by to tak byt. Nejak mi to vypadlo jak jsem presypaval nektery styly